src/jsMain/kotlin/IdentityProviderJS.kt

```diff
@@ -28,10 +28,10 @@ class IdentityProviderClientJS(region: String, clientId: String) {
                 .getOrThrow()
         }
 
-    fun resendConfirmationCode(username: String): Promise<CodeDeliveryDetailsJS> =
+    fun resendConfirmationCode(username: String): Promise<ResendConfirmationCodeResponseJS> =
```

Comment:
In that case we could directly return `CodeDeliveryDetailsJS`

Reply:
Why is that? All other functions return a ResponseJS object as far as I can see.

Reply:
Just thought it would be fine since it only stores the `CodeDeliveryDetails`. But yeah, might be better to leave it as is for consistency and to reflect the actual returned AWS object.